### ISSUE TITLE
Support ES6 config

### ISSUE THREAD

**Issue by derrickb** [PepsRyuu/nollup#40]
I don't know if there are any limitations preventing this, but it would be great to allow ES6 module imports/exports in `rollup.config.js`.

Eg:
```
import plugin from 'rollup-plugin';
export default {};
```
as opposed to:
```
const plugin = require('rollup-plugin');
module.exports = {};
```
I've just started using nollup and I need to rewrite my config so it's compatible.

**Comment by PepsRyuu** [comment]
I just had a look into how Rollup achieves this, and it's... an interesting solution... https://github.com/rollup/rollup/blob/master/cli/run/loadConfigFile.ts 

They use Rollup to compile the config file, and then override require to evaluate the code dynamically. Probably the same can be done in Nollup, don't think it would be too hard to add. I'll try to add this as soon as I can! :)

**Comment by PepsRyuu** [comment]
Small update: https://github.com/PepsRyuu/nollup/tree/ESMRollupConfigSupport

Just need to add test cases and tidy it up a bit. :)

**Comment by PepsRyuu** [comment]
Released in ```0.8.0```. Let me know if there's any issues!
